main.c

```diff
@@ -15,9 +15,51 @@ const char *SORT_NAMES[] = {"Merge Sort", "Quick Sort", "Bucket Sort"};
 // and return the number of inversions. Uncomment case 1 in InversionsHelper
 // function to test code. Compare your algorithms output to the correct # of
 // inversions
-//  int MergeSort(int arr[]) {
+ int MergeSort(int arr[], int start, int end) {
+   
+    int inversion_MergeSort = 0;
 
-// }
+    if (start < end){
+
+      int mid = start+(end-start)/2;
+
+      int pointer_left = start;
+      int pointer_right = mid+1;
+
+      inversion_MergeSort += MergeSortAndCount(a, pointer_left, mid);
+      inversion_MergeSort += MergeSortAndCount(a, pointer_right, end);
```

Comment:
The identifier 'a' is undeclared. Perhaps you forgot to declare this variable?

Reply:
oh, we use different name of the array variable "a" and"arr".